controllers/machinedeletionremediation_controller_test.go

```diff
@@ -135,26 +135,6 @@ var _ = Describe("Machine Deletion Remediation CR", func() {
 					Expect(underTest.GetAnnotations()).ToNot(BeNil())
 				})
 			})
-
-			When("machine deletion is not completed yet", func() {
```

Comment:
why did you remove this? :thinking:

Reply:
this test was possible before this rework, because
- Reconcile loop inferred the machine deletion was requested already from the presence of the annotation
- I could trick the Reconcile loop creating a CR with an annotation inside

With the rework, this is not the case anymore, Reconcile loop always checks for Machine's existence either from CR's annotation, or Node's annotation.

To make it work, I should set a deletiontimestamp into the machine, without actually request its deletion (otherwise the test is too fast deleting it and Reconcile loop does not have the time to see the machine "deleting" phase)

Reply:
ok 👍🏼